bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -12456,6 +12456,15 @@ public void setWorldPixelForWayPoints(final IntObjectHashMap<Point> worldPositio
       _twpWorldPosition.put(projectionHash + zoomLevel, worldPositions);
    }
 
+   public boolean temperaturesExist() {
```

Comment:
Because there are several temperatures in this case (average, min and max), I felt like the singular `is` didn't apply and as `areTemperatureValid` doesn't sound good, I decided to use apply [this recommendation](https://stackoverflow.com/a/1566778)
But considering your review, I have renamed to `isTemperatureAvailable`

Reply:
Programming code is not always readable like a prose. In this case I would prefer to keep all methods together which return a boolean value. It's mostly a compromise between both of these.

Reply:
Sure, I am also liking consistency across the code.
I think sometimes it's hard to find a guideline because not everything is consistent.

For example, I was using `TourData` today and found very confusing that it's not consistent:

[avgPulse](https://github.com/mytourbook/mytourbook/blob/bd51e10dbdd2a5883e489ecba5ad1fdd3cb20a1c/bundles/net.tourbook/src/net/tourbook/data/TourData.java#L414) is used but [powerAvg ](https://github.com/mytourbook/mytourbook/blob/bd51e10dbdd2a5883e489ecba5ad1fdd3cb20a1c/bundles/net.tourbook/src/net/tourbook/data/TourData.java#L583)is defined too
I would think it should be either `avgPulse` and `avgPower` or `pulseAvg` and `powerAvg` but not two different and in this case, i don't know which one follows the correct guideline

Reply:
`avgPulse` is from the beginning of MT, +15 years ago when there were not many fields in `TourData` but now there are so many fields that I try to "categorize" them -> `power_Avg`

Instead of starting a very big renaming action, with all it's possible risks, I mostly keep it as it is. The text keys in `messages.properties` also have 2 different naming rules which I also don't want to change, because it would need a lot of work to change it without any advantages for a normal user.

![image](https://user-images.githubusercontent.com/1283445/218945446-9cba9db7-b18f-4c7e-b882-3b2b0d3b684a.png)

Reply:
I get that some fields are older and it's not worth the risk to rename them after such a long time.

My point was that it makes it hard at times to know which rule/convention is to adopt at a given time and therefore can induce some mistakes

Reply:
Sometimes I also have to look on previous namings to decide how to name something new. With all of the invented names in MT I cannot remember all of them.